commons/src/main/java/org/apache/mesos/elasticsearch/common/cli/validators/CLIValidators.java

```diff
@@ -65,4 +65,18 @@ public void validate(String name, String value) throws ParameterException {
             }
         }
     }
+
+    /**
+     * Ensures that the string is not empty. Will strip spaces.
+     */
+    public static class ListOfSizeTwo implements IParameterValidator {
+        @Override
+        public void validate(String name, String value) throws ParameterException {
+
+            if (value.isEmpty() || !value.contains(",") || value.split(",").length != 2) {
```

Comment:
Yes it is required. The idea was that only two ports are valid. eg. "9200,9300", not "9200,9300,9400". So it must have a length of two.